### DISCUSSION TITLE
Location Installation Ungoogled Chromium on Windows

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#2267] [Q&A] [answered]
Hello...

Can i locate `ungoogled-chromium` installation?

Because, I prefer install an application in `Program Files` rather than `App Data\Local`.

Or

I should copy Chromium folder and move to that destination directory?

But, in future, if i update/install new `ungoogled-chromium` it will installed on `App Data\Local` again. So, i must use `.zip` version?

**Comment by PF4Public** [comment, marked as answer]
> So, i must use `.zip` version?

If you wish to unpack ungoogled chromium yourself, yes.

**Reply by ghost** [reply]
Yeah, it's okay. I have considered it and will use the `.zip` version. Thanks for your reply.
